docker-compose.yml

```diff
@@ -32,8 +32,10 @@ services:
       - spdashboard_mongo:/var/lib/mongo
     ports:
       - "443:443"
-    expose: 
-      - 3306
+      - "3306:3306"
+#    expose:
+#      - 3306
```

Comment:
Dev leftover? 

Either re-enable or remove if not needed anymore

Reply:
removed